packages/core/test/transient.test.ts

```diff
@@ -614,32 +614,27 @@ describe('transient states (eventless transitions)', () => {
     expect(() => actorRef.start()).not.toThrow();
   });
 
-  it('should be taken even in absence of other transitions', () => {
-    let shouldMatch = false;
-
+  it('should not be taken even in absence of other transitions', () => {
```

Comment:
Confused about the wording, isn't it taken?

Reply:
The wording is correct when it comes to the goal of this - but the assertions in this test were still wrong (the CI was failing because of that). I just pushed fixes for those tests and removed some old duplicates